source/_code.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
+          aria-expanded="false"
```

Comment:
It has a `display: none` which the js removes on setup. I believe that since screen readers ignore elements with a display of none, it's accomplishing the same thing?